src/awsexplorer/developerTools.ts

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as telemetry from '../shared/telemetry/telemetry'
+import { cdkNode, CdkRootNode } from '../cdk/explorer/rootNode'
+import { ResourceTreeDataProvider, TreeNode } from '../shared/treeview/resourceTreeDataProvider'
+import { once } from '../shared/utilities/functionUtils'
+
+export interface RootNode extends TreeNode {
+    canShow?(): Promise<boolean> | boolean
+    readonly onDidChangeVisibility?: vscode.Event<void>
+}
+
+const roots: readonly RootNode[] = [cdkNode]
```

Comment:
Nodes will need to be added here. I suppose it could also be a global but that can wait.